schedule/yast/select_modules_and_patterns+registration/select_modules_and_patterns+registration@s390x-zVM.yaml

```diff
@@ -15,40 +15,24 @@ vars:
   ADDONS: all-packages
```

Comment:
let's make consistent the name of the file using `...s390x_zvm` all lower letters and with underscore, could you please apply to the rest of the file and MR?

Reply:
OK

Reply:
OK